test_binding/main.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+const { platformFilename, loadDylib } = Deno;
+
+let localPath: any = import.meta.url.split("/");
+localPath.pop();
+localPath = localPath.join("/");
+
+const dLib = loadDylib(localPath + "/../target/release/" + platformFilename("test_binding"));
```

Comment:
What if it's a debug build? Can you somehow pass the path to the DLL through an env var?

Reply:
'DENO_BUILD_MODE'?

Reply:
It would be more robust if this test code didn't know anything about the build tree structure. I don't think DENO_BUILD_MODE is always defined. You'll have to pass the DLL path in through the caller. (Maybe necessitates a custom python test)

Reply:
Going to use `DL_PATH_TEST_BINDING` here, and I changed integration_tests.py to expose `DENO_BUILD_PATH` for the slight variation of this in tests.